deploy/vagrant/provision/playbook.yaml

```diff
@@ -25,7 +28,7 @@
     - name: configure hosts
       script: host.sh
     - name: rename hostname from TachyonXXX to TachyonXXX.local in virtualbox
-      shell: sudo hostname `hostname`.local
+      shell: sudo hostname `hostname | cut -d'.' -f1`.local # if provision multiple times, won't become TachyonXXX.local.local.local
```

Comment:
I should have set SPARK_MASTER_IP instead of changing hostname here, dirty.  Addressed in the following commit.